cddagl/ui/views/soundpacks.py

```diff
@@ -1144,50 +1144,54 @@ def game_dir_changed(self, new_dir):
         self.size_le.setText('')
         self.homepage_tb.setText('')
 
-        soundpacks_dir = os.path.join(new_dir, 'data', 'sound')
-        if os.path.isdir(soundpacks_dir):
-            self.soundpacks_dir = soundpacks_dir
+        soundpacks_dirs = [ 
+            os.path.join(new_dir, 'data', 'sound'),
+            os.path.join(new_dir,         'sound')  # User sound
+        ]
+        for soundpacks_dir in soundpacks_dirs:
+            if os.path.isdir(soundpacks_dir):
+                self.soundpacks_dir = soundpacks_dir
```

Comment:
`self.soundpacks_dir` should probably always remains in os.path.join(new_dir, 'data', 'sound'). It is used in move_new_soundpack for instance. With your code, it will be changed to os.path.join(new_dir, 'sound') after the loop.